gene2phenotype_project/gene2phenotype_app/views/search.py

```diff
@@ -63,7 +63,7 @@ def get_queryset(self):
                     base_phenotype & Q(lgdpanel__panel__name=search_panel) |
                     base_phenotype_2 & Q(lgdpanel__panel__name=search_panel) |
                     base_g2p_id & Q(lgdpanel__panel__name=search_panel)
-                ).order_by('locus__name', 'stable_id__stable_id').distinct()
+                ).order_by('locus__name', 'disease__name').distinct()
```

Comment:
Question - why are we ordering by disease name instead of stable id?

Reply:
This way the diseases with the same mechanism are displayed together. It looks more organised this way.